hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileWriterImpl.java

```diff
@@ -781,6 +783,11 @@ public Cell getLastCell() {
     return lastCell;
   }
 
+  @VisibleForTesting
```

Comment:
So many method exposes for the test-ability.   Are we really testing the FT way?  That a prevCell been passed and we pollute that cell's backing BB/byte[] after a shipped call?  That would be the best and in that I dont think we need to expose these many method.

Reply:
replace the UT with TestHFile#testDBEShipped